src/hooks/useClaimOrderCallback.ts

```diff
@@ -101,59 +102,87 @@ export function useGetAuctionProceeds(
   auctionIdentifier: AuctionIdentifier,
   derivedAuctionInfo: DerivedAuctionInfo,
 ): AuctionProceedings {
+  const { auctionDetails, auctionInfoLoading } = useAuctionDetails(auctionIdentifier)
   const { claimInfo } = useGetClaimInfo(auctionIdentifier)
+  const {
+    auctioningToken,
+    biddingToken,
+    clearingPriceOrder,
+    clearingPriceSellOrder,
+    clearingPriceVolume,
+  } = derivedAuctionInfo
 
   if (
     !claimInfo ||
-    !derivedAuctionInfo?.biddingToken ||
-    !derivedAuctionInfo?.auctioningToken ||
-    !derivedAuctionInfo?.clearingPriceSellOrder ||
-    !derivedAuctionInfo?.clearingPriceOrder ||
-    !derivedAuctionInfo?.clearingPriceVolume
+    !biddingToken ||
+    !auctioningToken ||
+    !clearingPriceSellOrder ||
+    !clearingPriceOrder ||
+    !clearingPriceVolume ||
+    auctionInfoLoading ||
+    !auctionDetails
   ) {
     return {
       claimableBiddingToken: null,
       claimableAuctioningToken: null,
     }
   }
-  let claimableAuctioningToken = new TokenAmount(derivedAuctionInfo?.auctioningToken, '0')
-  let claimableBiddingToken = new TokenAmount(derivedAuctionInfo?.biddingToken, '0')
+
+  let claimableAuctioningToken = new TokenAmount(auctioningToken, '0')
+  let claimableBiddingToken = new TokenAmount(biddingToken, '0')
+
+  const minFundingThresholdAmount = new TokenAmount(
+    biddingToken,
+    auctionDetails.minFundingThreshold,
+  )
+
+  const currentBiddingAmount = new TokenAmount(
+    biddingToken,
+    BigNumber.from(auctionDetails.currentBiddingAmount)
+      .mul(BigNumber.from(10).pow(biddingToken.decimals))
+      .toString(),
+  )
+
+  const minFundingReached = currentBiddingAmount.greaterThan(minFundingThresholdAmount)
+  const currentPrice = clearingPriceOrder.buyAmount.div(clearingPriceOrder.sellAmount)
+
+  if (!minFundingReached) {
+    for (const order of claimInfo.sellOrdersFormUser) {
+      const decodedOrder = decodeOrder(order)
+      claimableBiddingToken = claimableBiddingToken.add(
+        new TokenAmount(biddingToken, decodedOrder.sellAmount.toString()),
+      )
+    }
+
+    return {
+      claimableBiddingToken,
+      claimableAuctioningToken,
+    }
+  }
+
+  // Min funding is reached
+  // For each order from user add to claimable amounts (bidding or auctioning).
   for (const order of claimInfo.sellOrdersFormUser) {
     const decodedOrder = decodeOrder(order)
-    if (JSON.stringify(decodedOrder) == JSON.stringify(derivedAuctionInfo?.clearingPriceOrder)) {
+    if (JSON.stringify(decodedOrder) === JSON.stringify(clearingPriceOrder)) {
```

Comment:
if you could do the same in these 3 conditions when minimum is reached, it would be great.